components/consentRequestForm/dateInput/index.jsx

```diff
@@ -81,7 +71,7 @@ export default function DateInput(props) {
         classes={{ root: bem("date-button") }}
         type="button"
         // temporarily disabling until functionality to change date is available
-        disabled
+        // disabled
         aria-label="Set expiry date"
         onClick={() => setDatepickerOpen(!datepickerOpen)}
```

Comment:
I'd recommend lifting this up to `InputBase` via `endAdornment` so it's just an Icon, not a button — https://v4.mui.com/api/input-base/#inputbase-api

That way clicking on the date will open the calendar too.